app/assets/javascripts/tylium/behaviors.js.coffee

```diff
@@ -217,3 +221,8 @@ document.addEventListener "turbolinks:load", ->
   $('[data-behavior~=smooth-scroll]').on 'click', ->
     target = $(this).data('target');
     $('[data-id~=' + target + ']')[0].scrollIntoView({behavior: "smooth"});
+
+document.addEventListener "turbolinks:before-cache", ->
```

Comment:
I understand why we need this. I don't like it one bit, but I understand why.

Feels like we're often fighting turbolinks. I think the optimum solution would be for us to get the sidebar outside of tubolinks viewport entirely. That is outside the scope of this issue though.

Reply:
shouldn't you just save the state as part of the `click` handler?